docs/defaults.md

```diff
@@ -0,0 +1,26 @@
+## Default values
```

Comment:
This might be better suited in the `configuration.md` so people have a quick and easy point of reference for starting out?